_data/simple-icons.json

```diff
@@ -6987,6 +6987,11 @@
                 "type": "CC-BY-SA-4.0"
             }
         },
+        {
+            "title": "R3",
+            "hex": "EC1D24",
+            "source": "https://www.r3.com/contact-press-media/"
+        },
```

Comment:
Worth also adding the source link as the "guidelines" entry to the JSON - as there is an explicit 'guidelines' file available from that page.

Reply:
I have added the source link to brand guidelines.

Reply:
I think actually that both the source and guidelines entries should be the /contact-press-media/ page, as that has the logo pack for download, as well as the guidelines document. Any chance of tweaking that please @uiuxarghya? 🙏

Reply:
Okay I have updated the guidelines link.